src/napari_imagej/_module_utils.py

```diff
@@ -395,20 +398,48 @@ def _modify_function_signature(
         ]
         all_params = module_params + other_params
         function.__signature__ = sig.replace(
-            parameters=all_params, return_annotation=_return_type(module_info)
+            parameters=all_params, return_annotation=List[LayerDataTuple]
```

Comment:
Can we just remove the `return_annotation` here, and put the return type hint on `execute_module`?

Reply:
Fixed in 776413f

Reply:
Actually, we cannot. I had to reinstate the variable  return type in https://github.com/imagej/napari-imagej/commit/1f3b2e7753a7a826a710ef243922b4d3b71ceab7 :frowning: